### PR TITLE
CB-192: Add link to Cover Art Archive on /artist and /release-group pages

### PR DESCRIPTION
**Pull request by stasszczesniak**
Neither https://critiquebrainz.org/artist/467f69b6-8d6d-4d2c-bfad-e89adf2806ff nor https://critiquebrainz.org/release-group/1bd3ecb6-e064-3eef-a850-25eb5c0a50ee have any mention of the CoverArtArchive, but both pull in cover art from it.   
I've added some annotations about CAA on both pages.
[GCI task](https://codein.withgoogle.com/dashboard/task-instances/6387195350876160/)
[#CB-192](http://tickets.musicbrainz.org/browse/CB-192)

### CODE REVIEW

File: critiquebrainz/frontend/templates/artist.html [hunk 1]
```diff
@@ -159,6 +159,13 @@
       <div class="favicon-container"><img src="{{ url_for('static', filename='favicons/musicbrainz-16.ico') }}" /></div>
       <a href="//musicbrainz.org/artist/{{ artist.id }}"><em>{{ _('Edit on MusicBrainz') }}</em></a>
     </div>
+    <div id="CoverArtAnnotation">
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/artist.html]: Please format IDs consistently. `cover-art-annotation`

File: critiquebrainz/frontend/templates/artist.html [hunk 2]
```diff
@@ -159,6 +159,13 @@
...
+      <span class="text-muted">
+        Covers provided by :
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/artist.html]: No space before colon.

File: critiquebrainz/frontend/templates/artist.html [hunk 3]
```diff
@@ -159,6 +159,13 @@
...
+        <a href="https://coverartarchive.org/">CoverArtArchive</a>
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/artist.html]: > Cover Art Archive

**Comment by stasszczesniak** [comment]
Ok, it should be fine now.

File: critiquebrainz/frontend/templates/artist.html [hunk 4]
```diff
@@ -159,6 +159,13 @@
       <div class="favicon-container"><img src="{{ url_for('static', filename='favicons/musicbrainz-16.ico') }}" /></div>
       <a href="//musicbrainz.org/artist/{{ artist.id }}"><em>{{ _('Edit on MusicBrainz') }}</em></a>
     </div>
+    <div id="cover-art-annotation">
+      <span class="text-muted">
+        Covers provided by :
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/artist.html]: Should be no space before colon.

File: critiquebrainz/frontend/templates/release_group.html [hunk 5]
```diff
@@ -45,7 +45,14 @@ <h2 class="pull-left">
           .on("error", function () { $(this).attr("src", "/static/img/missing-art.png"); })
           .attr("src", "//coverartarchive.org/release-group/{{ id }}/front-250");
     </script>
-    <div id="spotify-play-button">
+    <div id="cover-art-annotation">
+      <span class="text-muted">
+	Cover provided by :
```
Comment (gentlecat) [inline critiquebrainz/frontend/templates/release_group.html]: [Same](https://github.com/metabrainz/critiquebrainz/pull/12/files#r47434279) here.

File: critiquebrainz/frontend/templates/artist.html (hunk 4 above)
Comment (gentlecat) [inline critiquebrainz/frontend/templates/artist.html]: Better wording might be "Cover art provided by Cover Art Archive." No need to use colons then. CAA is our only provider.

**Pull request closed without merging**